change/@ni-nimble-components-330f85ba-9862-4e5a-8422-20d7a7143a1e.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Thanks for the great detailed PR description! But one thing is missing: how much did your changes improve performance? I'm dying to know!

Reply:
Here is a short breakdown on the benchmarking I did today:
- when rendering dies without the text the difference is not really significant based on the times I obtained(the method in the PR performed slightly better though).

| Data Points  | Current Render Time (ms) | Improved Render Time (ms) |
| ------------- | ------------- |------------- |
| 1k  | 51.70  | 49.70 |
| 10k  | 92.20  | 82.80 |
| 100k  | 453.80 | 402.19 |
| 200k  | 835.40  | 764.59 |
| 500k  | 2278.19  | 1851.30 |

When it comes to rendering dies and their values the difference is significant:

| Data Points  | Current Render Time (ms) | Improved Render Time (ms) |
| ------------- | ------------- |------------- |
| 1k  | 55.70  | 51.70 |
| 10k  | 153.19  | 126.09 |
| 100k  | 1412.90 | 812.80 |
| 200k  | 4496.80  | 1629.19 |
| 500k  | 11538.19  | 4227.69 |

Please note that these are the ~average values obtained after the page was refreshed. It's important how fast the page can re-render when the values are displayed, because this happens usually when the user zooms in on the wafer-map. Whilst zoomed in, the only way to navigate is by dragging the wafer-map, and that means that the dies will be re-rendered. I also observed a slight increase in the FPS when dealing with larger datasets (I measured this by using Chrome's built in dev tool).

Reply:
I resolve this comment. I posted the table above into the description of the PR.